modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -655,21 +660,24 @@ services:
     frequencyTimeUnit: MINUTES
 
 ----
-The `/security/cacerts.jks` truststore file is imported into the `$MULE_HOME/conf` folder and renamed as `anypoint-truststore.jks`. +
++
+This command imports the `/security/cacerts.jks` truststore file into the `$MULE_HOME/conf` folder and renames it  `anypoint-truststore.jks`.
++
 [NOTE]
-The default truststore was renamed `anypoint-truststore.jks` in Runtime Manager agent version 1.10.0 and later. In earlier versions of Runtime Manager agent, the default truststore was named `truststore.jks`. 
-
-. Restart the Mule runtime engine, and verify the Runtime Manager agent REST interface starts up successfully. Add SSL debugging to the Mule runtime engine logging. `./mule -M-Djavax.net.debug=all`
-
-
-=== Configuring a Mule Runtime for Two-Way TLS with the Mule Runtime managed by Anypoint Control Plane
+In Runtime Manager agent version 1.10.0 and later, the name of the default truststore was `anypoint-truststore.jks`. In earlier versions of Runtime Manager agent, the name was `truststore.jks`.
 
-Here is an example of configuring two-way TLS with the Mule Runtime connected to Anypoint Control Plane.
+. Restart Mule runtime engine and verify that the Runtime Manager agent REST interface starts successfully.
+. Add SSL debugging to the Mule runtime engine logging:
++
+`./mule -M-Djavax.net.debug=all`
 
-The steps to configure TLS are:
+=== Configure Two-Way TLS for Mule Runtime Engine Managed by Control Plane
 
-. Generate a keystore (public/private key pair) to identify the Runtime Manager agent (server). Set the CN to match the Runtime Manager agent's hostname or IP Address.
+To configure two-way TLS when Mule is connected to the control plane:
 
+. Generate a keystore (public/private key pair) to identify the Runtime Manager agent (server).
```

Comment:
ditto re. pair name
Please search for and change GLOBALLY. I will stop marking from here to the end of this request.

Reply:
Good. I already s+r :)